Assets/HoloToolkit/Utilities/Scripts/Attributes/DropDownGameObjectAttribute.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text.RegularExpressions;
+using UnityEngine;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace HoloToolkit.Unity
+{
+    // Displays a drop-down menu of GameObjects that are limited to the target object
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]
+    public sealed class DropDownGameObjectAttribute : DrawOverrideAttribute
```

Comment:
I'm pretty sure all these attributes scripts could be in an Editor folder and we can lose the `UNITY_EDITOR` declarations

Reply:
These attributes are being used in scripts outside of Editor, you'd have to wrap the attributes themselves in #if UNITY_EDITOR declarations to prevent compile errors.